scripts/setup-bun.ps1

```
@@ -0,0 +1,7 @@
+$ErrorActionPreference = 'Stop'
+
+Invoke-RestMethod https://bun.sh/install.ps1 | Invoke-Expression
```

PSScriptAnalyzer (1.25.0)

[warning] Invoke-Expression is used. Please remove Invoke-Expression from script and find other options instead.

(PSAvoidUsingInvokeExpression)